<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>capacitor_page</class>
 <widget class="QDialog" name="capacitor_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_capacitance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacitance</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_voltage">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tempco">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature Coefficient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_capacitance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_units">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_voltage">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tempco">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
